src/commands/login.js

```diff
@@ -38,11 +38,16 @@ module.exports = new Command("login")
       {
         type: "confirm",
         name: "collectUsage",
-        message: "Allow Firebase to collect anonymous CLI usage and error reporting information?",
+        message: "Allow Firebase to collect CLI usage and error reporting information?",
       },
     ])
       .then(function() {
         configstore.set("usage", options.collectUsage);
+        if (options.collectUsage) {
+          utils.logBullet(
```

Comment:
Should this happen _before_ we ask? This seems like information that is useful before the choice, not after...

Reply:
Done.